CONTRIBUTING.md

```
@@ -378,152 +466,104 @@ Use the feature request template and include:
-#### Fork-First Development Process
+### What lives in this repository
 
-We recommend a **fork-first development approach** for applications to ensure quality and long-term maintainability:
+Example and sample code is under **`contracts/examples/`**, not a top-level `applications/` directory. Per [LICENSE](LICENSE), **`contracts/examples/` is excluded from MPL-2.0**; each file declares its own license (today’s in-repo examples use **`SPDX-License-Identifier: MIT`**).
 
-##### Phase 1: Fork Development
-1. **Fork the repository** to your own GitHub account
-2. **Develop your application** in your fork
-3. **Maintain long-term** in your fork
-4. **Iterate and improve** based on real-world usage
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)